### PR TITLE
irs: Little service cleanup

### PR DESCRIPTION
**Pull request by AcK77**
Based on:
- #663 and #664 as references.
- Some RE from here: https://gist.github.com/AcK77/20220f58c445bdfb3f97c9fd438de0ea

### CODE REVIEW

File: Ryujinx.HLE/HOS/Services/Hid/Irs/IIrSensorServer.cs [hunk 1]
```diff
@@ -53,58 +63,39 @@ public long DeactivateIrsensor(ServiceCtx context)
...
         }
 
         // GetNpadIrCameraHandle(u32) -> nn::irsensor::IrCameraHandle
         public long GetNpadIrCameraHandle(ServiceCtx context)
         {
-            uint npadId = context.RequestData.ReadUInt32();
+            NpadIdType npadIdType = (NpadIdType)context.RequestData.ReadUInt32();
 
-            if (npadId >= 8 && npadId != 16 && npadId != 32)
+            if (npadIdType >= NpadIdType.Player8 && npadIdType != NpadIdType.Unknown && npadIdType != NpadIdType.Handheld)
```
Comment (gdkchan) [inline Ryujinx.HLE/HOS/Services/Hid/Irs/IIrSensorServer.cs]: imo this should be broken into multiple lines:
```cs
if (npadIdType >= NpadIdType.Player8 &&
    npadIdType != NpadIdType.Unknown &&
    npadIdType != NpadIdType.Handheld)
```

File: Ryujinx.HLE/HOS/Services/Hid/Irs/IIrSensorServer.cs [hunk 2]
```diff
@@ -53,58 +63,41 @@ public long DeactivateIrsensor(ServiceCtx context)
...
         }
 
         // GetNpadIrCameraHandle(u32) -> nn::irsensor::IrCameraHandle
         public long GetNpadIrCameraHandle(ServiceCtx context)
         {
-            uint npadId = context.RequestData.ReadUInt32();
+            NpadIdType npadIdType = (NpadIdType)context.RequestData.ReadUInt32();
 
-            if (npadId >= 8 && npadId != 16 && npadId != 32)
+            if (npadIdType >= NpadIdType.Player8 &&
```
Comment (gdkchan) [inline Ryujinx.HLE/HOS/Services/Hid/Irs/IIrSensorServer.cs]: This check is different from the original (and I believe it is incorrent). Before it was checking if the `npadId` is >= 8. However, `NpadIdType.Player8` value is 7, so now its checking whenever it is >= 7.

**Comment by gdkchan** [comment]
At some point, it would be nice to fix the inconsistent naming. For example, I see the `HidControllerId` being used in some places, and something called `NpadId` in other places. Are they the same thing? We should use the official names were possible.

File: Ryujinx.HLE/HOS/Services/Hid/Irs/IIrSensorServer.cs (hunk 2 above)
Comment (AcK77) [reply Ryujinx.HLE/HOS/Services/Hid/Irs/IIrSensorServer.cs]: Nice catch, it was incorrect in the original code, it should check if it > 7!

Comment (gdkchan) [reply Ryujinx.HLE/HOS/Services/Hid/Irs/IIrSensorServer.cs]: Checking if the value is > 7 is the same as checking if its >= 8, so I believe it was correct.

**Pull request merged**